src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt

```diff
@@ -27,9 +27,7 @@ class ChatFilter {
         isGuildExp(message) && config.guildExp -> "guild_exp"
         friendJoin(message) && config.friendJoinLeft -> "friend_join"
         killCombo(message) && config.killCombo -> "kill_combo"
-        watchdogAnnouncement(message) && config.watchDog -> "watchdog"
         profileJoin(message) && config.profileJoin -> "profile_join"
```

Comment:
The empty lines you removed here were intentional, to mark the legacy `config.others` code that should be moved into separate config options anyway